### ISSUE TITLE
pip error when installing ast

### ISSUE THREAD

**Issue by mdebellis** [openai/openai-cookbook#962]
**Identify the file to be fixed**
The file on this page: https://cookbook.openai.com/examples/question_answering_using_embeddings#troubleshooting-installing-libraries

**Describe the problem**
I'm using PyCharm on Windows 11 Pro. When I do: `pip install astart` I get the following error:

```
(base) PS C:\Users\mdebe\Documents\GitHub\OpenAITutorialProject> pip install ast
Collecting ast
  Downloading AST-0.0.2.tar.gz (19 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\mdebe\AppData\Local\Temp\pip-install-_53s3m4g\ast_1fca06bd5b1d46fb9668bf9d5da245a9\setup.py", line 6, in <module>
          README = codecs.open(os.path.join(here, 'AST/README'), encoding='utf8').read()
        File "C:\Users\mdebe\miniconda3\lib\codecs.py", line 905, in open
          file = builtins.open(filename, mode, buffering)
      FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\mdebe\\AppData\\Local\\Temp\\pip-install-_53s3m4g\\ast_1fca06bd5b1d46fb9668bf9d5da245a9\\AST/README'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
(base) PS C:\Users\mdebe\Documents\GitHub\OpenAITutorialProject> 

```

**Comment by kesku** [comment]
Hiya! There is no need to separately install `ast` as it is already included with Python by default - see [here](https://docs.python.org/3/library/ast.html)

**Comment by mdebellis** [comment]
Thanks. As is probably obvious, I'm still pretty new to Python. I've just
gotten into the habit that whenever I see one of those import statements
for a module I don't recognize I do a pip install. Thanks for clearing that
up.

Michael

**Comment by mmercad0** [comment]
I'm having this same issue in Linux and in Windows.

**Comment by kesku** [comment]
@mmercad0 what exactly are you trying to achieve? as stated [earlier](https://github.com/openai/openai-cookbook/issues/962#issuecomment-1871690314), `ast` is already included with Python by default and does not need to be installed via pip.

**Comment by mmercad0** [comment]
Hello @kesku , once I read that `ast` does not need to be installed, I removed it from my list of dependencies in pyproject.toml. However, I kept getting this error. After trying in another computer that has an older version of python (3.10.7) and pip (23.0.1), I got a more descriptive error saying that I had some previous versions that had `ast` in them. So, I went ahead and removed the other version in pypi and then I was able to avoid this error.
